docs/sw-update-conf.json

```diff
@@ -1,123 +1,156 @@
-    "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
-        }
+  "modules": {
+    "octoprint": {
+      "type": "github_release",
+      "develop": {
+        "type": "github_commit"
       }
-    }
-  },
-  "mrbeamdoc": {
-    "name": "Mr Beam Documentation",
-    "repo": "MrBeamDoc",
-    "pip": "https://github.com/mrbeam/MrBeamDoc/archive/{target_version}.zip",
-    "restart": "octoprint"
-  },
-  "netconnectd": {
-    "name": "OctoPrint-Netconnectd Plugin",
-    "repo": "OctoPrint-Netconnectd",
-    "pip": "https://github.com/mrbeam/OctoPrint-Netconnectd/archive/{target_version}.zip",
-    "restart": "octoprint",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
+    },
+    "mrbeam": {
+      "name": " MrBeam Plugin",
+      "repo": "MrBeamPlugin",
+      "restart": "environment",
+      "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
+      "develop": {
+        "update_folder": "/tmp/octoprint/mrbeamplugin",
+        "update_script_relative_path": "scripts/update_script.py",
+        "update_script": "{{python}} '{update_script}' --branch={{branch}} --force={{force}} '{{folder}}' {{target}}",
+        "methode": "update_script"
+      },
+      "alpha": {
+        "update_folder": "/tmp/octoprint/mrbeamplugin",
+        "update_script_relative_path": "scripts/update_script.py",
+        "update_script": "{{python}} '{update_script}' --branch={{branch}} --force={{force}} '{{folder}}' {{target}}",
+        "methode": "update_script"
+      },
+      "dependencies": {
+        "mrbeam-ledstrips": {
+          "repo": "MrBeamLedStrips",
+          "pip": "https://github.com/mrbeam/MrBeamLedStrips/archive/{target_version}.zip",
+          "global_pip_command": true,
+          "beamos_date": {
+            "2021-06-11": {
+              "pip_command": "sudo /usr/local/mrbeam_ledstrips/venv/bin/pip"
+            }
+          }
+        },
+        "iobeam": {
+          "repo": "iobeam",
+          "pip": "git+ssh://git@bitbucket.org/mrbeam/iobeam.git@{target_version}",
+          "global_pip_command": true,
+          "beamos_date": {
+            "2021-06-11": {
+              "pip_command": "sudo /usr/local/iobeam/venv/bin/pip"
+            }
+          }
+        },
+        "mrb-hw-info": {
+          "repo": "mrb_hw_info",
+          "pip": "git+ssh://git@bitbucket.org/mrbeam/mrb_hw_info.git@{target_version}",
+          "global_pip_command": true,
+          "beamos_date": {
+            "2021-06-11": {
+              "pip_command": "sudo /usr/local/iobeam/venv/bin/pip"
+            }
+          }
+        },
+        "mrbeamdoc": {
+          "repo": "MrBeamDoc",
+          "pip": "https://github.com/mrbeam/MrBeamDoc/archive/{target_version}.zip"
```

Comment:
no need to add beamos_date?

Reply:
The beamos date is only needed if you need a seperate config for the given date